### PR TITLE
Add short description of Streamz to README.rst

### PR DESCRIPTION
**Pull request by quipa**
Add short description to inform users that find the package through the GitHub repository page.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,7 +3,11 @@ Streamz
 
 |Build Status| |Doc Status| |Version Status|
 
-See documentation at `streamz.readthedocs.org <https://streamz.readthedocs.org>`_.
+Streamz helps you build pipelines to manage continuous streams of data. It is simple to use in simple cases, but also supports complex pipelines that involve branching, joining, flow control, feedback, back pressure, and so on.
```
Comment (CJ-Wright) [inline README.rst]: "and more"?

Comment (quipa) [reply README.rst]: Sounds good too me! I just copy pasted from readthedocs ;).

**Comment by kivo360** [comment]
Why isn't this merged in?

**Comment by martindurant** [comment]
@mrocklin , were there any comments or issues with this PR, or was it not merged just because of failing tests, which ought to be passing now? (may require merge from master)

**Comment by martindurant** [comment]
I would be happy to fetch from this fork, merge and re-PR myself, even edit the text if required.

**Pull request merged**